### ISSUE TITLE
Usage by other small aircraft?

### ISSUE THREAD

**Issue by yohanboniface** [UAVTracking/UAVTrackingProtocol#2]
There has been a discussion about whether this protocol should also be used by other small aircraft (gliders, small planes…).
The rationale is that they share the lower airspace.

Among the questions:
- which field to define the aircraft category? And which values?
- should each type of aircraft transmit on their own frequency (eg. 868.4 for UAVs, 868.2 for gliders…) ?

**Comment by snip** [comment]
I think collision avoidance have to be enable for all flying UAV or aircraft in same airspace.
So this UAV protocol have to be usable by already existing small aircrafts & gliders.

**Comment by snip** [comment]
Current situation is that lot of small aircraft and gliders are equipped with [Flarm](http://flarm.com/) collision avoidance devices. (Flarm is mandatory for gliders in some countries like France).

So it can be very interesting the protocol used by UAV is compatible with Flarm.

But Flarm radio protocol is closed. But maybe this new UAV protocol can be at least hardware compatible with Flarm devices. So Flarm can updae their software to be able to receive this protocol informations.

It can also be interesting that UAV can receive Flarm information (for UAV pilot or autopilot). But this reuire Flarm to transmit on an open protocol protocol.

**Comment by ClLTL** [comment]
Many standardization groups are currently working, in Europe and in the USA, on UAV versus aircraft conflict issues. One important point that has to be considered is that the rules of the air prevail in any conflict. ICAO annex II states that "An aircraft shall not be operated in such proximity to other aircraft as to create a collision hazard." (section 3.2.1) meaning that avoiding a collision is not enough. Any aircraft has to remain "well clear" of other traffic. If the protocol used by UAV is compatible with FLARM, FLARMed aircraft's pilot will get a good situation awareness to pass well clear of UAV! (and vice versa).

**Comment by darekzbik** [comment]
In my opinion splitting to various type of aircrafts is not good idea. It makes receiving signals from all possible threats require frequency hopping which lead to losing data in other channels. If we would like to avoid congestion in the same area it is better to split airspace for regions (vertically or in a different way). By the way mentioned ISM frequencies are not allowed for broadcasting from the air so if you would like to do it in a proper way you should try a band which can be used without legal issues.

**Comment by yohanboniface** [comment]
> By the way mentioned ISM frequencies are not allowed for broadcasting from the air so if you would like to do it in a proper way you should try a band which can be used without legal issues.

I've heard this argument, and the opposite of this also. Can you elaborate on the topic and if possible source it so we make light on that topic? Thanks :)

**Comment by optimaltracking** [comment]
Hello. Why is broadcasting from the air not allowed?

**Comment by darekzbik** [comment]
Technically because broadcasted signal will cover huge are.
And more important see [FREQUENCY ALLOCATION TABLE](http://www.erodocdb.dk/docs/doc98//official/pdf/ERCREP025.PDF). Jump to page 100/280. Band 862 MHz - 890 MHz is intened for _BROADCASTING, FIXED,
MOBILE EXCEPT AERONAUTICAL MOBILE_ . I know that in many places it is simply ignored and both OGN and FLARM are using the same frequency but it should be solved in some way. The best approach will be finding a proper separate frequency just for drone tracking. If you guys have enough power to find a better frequency try to do it. This ISM band 868.x become more and more noisy all the time.

**Comment by pjalocha** [comment]
For channel capacity and diversity reason it would be good to use both frequencies: 868.2 and 868.4MHz. I can quote at least one case of the EPMR receiver where one of these frequencies it taken out by a strong nearby signal (most likely illegal...). Even if all participants transmit and listen randomly on the two frequencies, each would hear on average half of the packets of any other participant which is enough data for collision avoidance. And the ground receiver will hear all the packets thus from the tracking point of view we will have double channel capacity.
The transmissions on the two channels will interfere a little bit, thus the sensitivity will suffer a bit but still there would be a major improvement.

**Comment by pjalocha** [comment]
See my proposal for the Header: the first 32-bit word of the packet: on page 7-8: https://docs.google.com/document/d/1lx5d6qcAk7sZULNXXTvC7bQsNCZKk73gKW74jdlIMA4/edit#

I have there 1-bit value "Human", after suggestion of Alvaro, who reminded us that humans should have priority over drones ! :-)

There is as well a spare bit which could be used for additional indications.

**Comment by darekzbik** [comment]
In one of previous comments @ClLTL refer to some regulations:
>Many standardization groups are currently working, in Europe and in the USA, on UAV versus aircraft conflict issues. One important point that has to be considered is that the rules of the air prevail in any conflict. ICAO annex II states that "An aircraft shall not be operated in such proximity to other aircraft as to create a collision hazard." (section 3.2.1) meaning that avoiding a collision is not enough. Any aircraft has to remain "well clear" of other traffic. If the protocol used by UAV is compatible with FLARM, FLARMed aircraft's pilot will get a good situation awareness to pass well clear of UAV! (and vice versa).

According to [ICAO Annex2](https://www.icao.int/Meetings/anconf12/Document%20Archive/an02_cons%5B1%5D.pdf) powered aircraft should give a way to **airships**, **gliders** and **ballons**

>3.2.2.3 Converging. When two aircraft are converging at approximately the same level, the aircraft that has the other on its right shall give way, except as follows:
>a) power-driven heavier-than-air aircraft shall give way to airships, gliders and balloons;
>b) airships shall give way to gliders and balloons;
>c) gliders shall give way to balloons;
>d) power-driven aircraft shall give way to aircraft which are seen to be towing other aircraft or objects. 

So this discussion should be rather replace to 'how drones can receive positions of  **airships**, **gliders** and **ballons**. And drones should give a way.

Aslo as @pjalocha mentioned above humans should have priority over drones in general.

**Comment by optimaltracking** [comment]
I agree on your proposal but it is not possible with low price aeromodel and drones. Your proposal means to include a complex system of "detect and avoid". It will be possible only on professional drones.

**Comment by darekzbik** [comment]
It is law and size of drone does not changing it. Of course I'd like to be
aware of drones but still drones should give me a way. It is usually not a
problem for classic RC models because pilots of such models constantly look
around.

**Comment by pjalocha** [comment]
If drones use same position reporting protocol as gliders and other aircrafts flying in "low space" they would be made aware of other traffic. Autopilots like ArduPilot have already features like zones to avoid, etc. thus making them avoid other traffic is not impossible, it just needs some research and programming work. I understand drones want to be aware of other drones and to avoid them, thus this could be simply the same protocol and algorithm or some variants of it.

I should add that for example in the MAVlink protocol, already exist messages about collision: http://mavlink.org/messages/common thus this the ArduPilot foresee already detections and ways to avoid collisions.

**Comment by ClLTL** [comment]
@darekzbik 
Yes, you are right, the rules of the air are such that drones should give way to many other aircraft.  On top of that, it is important to keep in mind that the level of safety that is currently achieved in the airspace relies on the fact that when an aircraft is the priority one in an air-to-air conflict, he can however maneuver if the conflicting aircraft does not see him. If the conflicting aircraft is a small drone, this mitigation does not work as the drone may be seen too late by the manned aircraft's pilot. Small drones are making the detect and avoid principle  non symmetric any more, they should have a higher performance to detect and avoid other traffic... even if the 10€ cost target is not met, safety first!
